site/en/docs/devtools/console/reference/index.md

```diff
@@ -267,6 +267,17 @@ To search for text in log messages:
 This section contains features related to running JavaScript in the Console. See [Run
 JavaScript][28] for a hands-on walkthrough.
 
+### Copy options {: #copy-options }
```

Comment:
These options are for string outputs. There are other types of content, objects, for example. They have different copy options. So, technically, this headline should either include all options for all types of outputs, or be just for strings.